ceph/ceph.py

```diff
@@ -766,6 +766,8 @@ def setup_packages(
         """
         if not build:
             build = self.rhcs_version
+        if self.ansible_config.get("use_cdn"):
```

Comment:
This might not required. `use_cdn` at test module should take care instead handling in lib modules.

Reply:
need to check regression here, if it is not working.